src/lib_ccx/file_functions.c

```diff
@@ -7,6 +7,9 @@ long FILEBUFFERSIZE = 1024*1024*16; // 16 Mbytes no less. Minimize number of rea
 #ifdef _WIN32
 WSADATA wsaData = {0};
 int iResult = 0;
+int isWin = 1;
```

Comment:
Dont make it dynamic for finding windows or linux, if code is windows specific then at compile time only include or exclude the code. Use #ifdef instead of isWin

Reply:
I had a dynamic toggle for this since in windows also there are only a the case of source multicast where this is needed, so instead of nested if inside ifdef I reduced code space. Never the less, I'll do that.